pkg/operator/profilecalculator.go

```diff
@@ -243,7 +243,7 @@ func (pc *ProfileCalculator) calculateProfile(nodeName string) (string, map[stri
 
 		if recommendAll[iStop].Priority == nil || recommendAll[i].Priority == nil {
 			// This should never happen as Priority is a required field, but just in case -- we don't want to crash below.
-			klog.Warningf("one or both of profiles %s/%s have undefined priority", recommendAll[iStop].Profile, recommendAll[i].Profile)
+			klog.Warningf("one or both of profiles %d/%d have undefined priority", recommendAll[iStop].Profile, recommendAll[i].Profile)
```

Comment:
This change is not correct, but it highlights an issue in the code. %s/%s is intended, but `recommendAll[iStop].Profile` and `recommendAll[i].Profile` are pointers to a string. They are both required values, so dereferencing them shouldn't result in a crash. But the Priority is also a required value.  I'd say we either we want to keep %s/%s and dereference recommendAll[iStop].Profile and recommendAll[i].Profile or remove the if clause completely, because Priority is also required.

Reply:
If the change is not directly related to the bump then we can do it in another PR.

Reply:
I think it is, since it stems from the klog bump

Reply:
Thanks for pointing this out, will fix by dereferencing the pointers.